File: app/src/main/java/com/jesusc24/xroadsthroughthecastle/ui/guia/GuiaFragment.java
package com.jesusc24.xroadsthroughthecastle.ui.guia;

import android.app.Fragment;
import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;

import androidx.annotation.Nullable;

public class GuiaFragment extends Fragment {

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        String url = "http://sru-wiki.achisoft.net/tiki-index.php";
        Uri uri = Uri.parse(url);
        Intent intent = new Intent(Intent.ACTION_VIEW, uri);
        startActivity(intent);
        //hacerlo desde un broadcast receiver
    }
}

File: app/src/main/java/com/jesusc24/xroadsthroughthecastle/ui/guia/GuiaFragment.kt
package com.jesusc24.xroadsthroughthecastle.ui.guia

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.jesusc24.xroadsthroughthecastle.data.constantes.Constants
import com.jesusc24.xroadsthroughthecastle.data.repository.UserRepository
import com.jesusc24.xroadsthroughthecastle.databinding.FragmentGuiaBinding
import com.jesusc24.xroadsthroughthecastle.utils.PreferencesManager

class GuiaFragment : Fragment() {

    private lateinit var binding: FragmentGuiaBinding

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        binding = FragmentGuiaBinding.inflate(layoutInflater)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        binding.webView.loadUrl("http://sru-wiki.achisoft.net/tiki-index.php")
    }

    override fun onResume() {
        super.onResume()
        val preferenceManager = PreferencesManager(context)
        preferenceManager.putBoolean(Constants.KEY_AVAILABILITY, true)
    }

    override fun onPause() {
        super.onPause()
        val preferenceManager = PreferencesManager(context)
        preferenceManager.putBoolean(Constants.KEY_AVAILABILITY, true)
    }

}